### ISSUE TITLE
[FluxStore Pro] - Problems & Requests

### ISSUE THREAD

**Issue by bikostudio** [inspireui/support#2642]
Hi, 

Unfortunately I'm having many issues with this App mentioned in Support ticket #2439. 

Not all of them answered yet specially the App building error on 1.6.6b. So I will continue with 1.6.5c becouse I don't have the App building issue with it.  

In this new ticket  will concentrate on the most important issues I want to fix  as I want to publish my Woocommerce App as soon as possible.

Here is my environment's  details:

Windows 8.1
Android Studio Latest Version
Flutter Latest Version
All Flutter and Dart plugins were installed
MStore API Plugin installed

Flutter doctor : 
![2020-04-10_183233](https://user-images.githubusercontent.com/62074493/79006535-ac39db00-7b59-11ea-82dd-fad6f6eccf71.jpg)

===============  Proof of my purchased   =====================

![2020-04-09_143006](https://user-images.githubusercontent.com/62074493/79005762-e4d8b500-7b57-11ea-9a99-09054b92eb98.jpg)

**Comment by bikostudio** [comment]
================ My Issues ===============================

==[ 1 ] ======== Error When Creating New Account =======

I was be able to create new account through the app and it creates an account on Woocommerce site as expected. But now I have a new problem I did not face before upgrading to 1.6.5c.

 First I got this error : 
![2020-04-10_095526](https://user-images.githubusercontent.com/62074493/79006721-0c308180-7b5a-11ea-8370-3c0e73c1a8b0.jpg)

I Realized that the Firebase API changed and I download the new google services json file and changed the API in all places in app it should be. 

Then I got this error 

![2020-04-10_183740](https://user-images.githubusercontent.com/62074493/79006902-5f0a3900-7b5a-11ea-8b01-e32c394216e2.jpg)

With this error line : 
I/BiChannelGoogleApi( 6634): [FirebaseAuth: ] getGoogleApiForMethod() returned Gms: com.google.firebase.auth.api.internal.zzaq@61e452d
I/flutter ( 6634): [Fluxstore][Resistration] PlatformException(FirebaseException, An internal error has occurred. [ Requests to this API identitytoolkit method google.cloud.identitytoolkit.v1.AuthenticationService.SignUp are blocked. ], null)

So I enabled the email AuthenticationService on Firebase. and after that the new user account only created on Firebase but not created on my Woocommerce website and still got this error.

![2020-04-10_183740](https://user-images.githubusercontent.com/62074493/79006902-5f0a3900-7b5a-11ea-8b01-e32c394216e2.jpg)

=====================  END =================================
==========================================================

**Comment by bikostudio** [comment]
==[ 2 ] ======== Error When Creating New Account =======

I have created a Firebase account with all settings by following your tutorial : https://docs.inspireui.com/fluxstore/firebase/

I can send test notifications to the test device now but when I tap on the notification on my phone it open the application. 


- It is not saved in Notify Messages page in the App.

- How to redirect user to Notify Messages page after tapping on the notification?


![2020-03-31_115054](https://user-images.githubusercontent.com/62074493/79007385-5e25d700-7b5b-11ea-9994-de27d4b9190c.jpg)

![index3](https://user-images.githubusercontent.com/62074493/79007413-6aaa2f80-7b5b-11ea-9d5a-e2949eb69392.jpg)

![index2](https://user-images.githubusercontent.com/62074493/79007430-7269d400-7b5b-11ea-962e-4a7a14637e97.jpg)


=======================  END =================================
============================================================

**Comment by bikostudio** [comment]
### **==[ Request 1 ] ======== Adding New Fields In Registration  =======**

I want to add a Company and a Mobile fields to the registration form also to be added to the WordPress profile as well.

**Comment by bikostudio** [comment]
==[ Request 2 ] ======== Get the location google map link in the order's page   =======

Till now the geocoding location is working fine getting the user location and Auto fill the address fields.
When user make a new order the new order saved to Woocommerce ----> orders section with the address by letters.
My question Is there a way to get the location map URL as will to be saved in order's page so admin can open it on google map?

**Comment by philipvu30** [comment]
Hi,

**Request 1 and 2:**  It's possible to add these features but currently we do not provide customization service.

For the above issues: 
- Please run command "flutter clean" then send me your project to this email: toan@inspireui.com. I'll help you fixing them. 

Thanks

**Comment by bikostudio** [comment]
Hi,

Finally, and after many tries, I got the 1.6.6b to work.



Now, I have some issues



Issue 1:

I have created a Firebase account with all settings by following your
tutorial : https://docs.inspireui.com/fluxstore/firebase/

I can send test notifications to the test device now but when I tap on the
notification on my phone it open the application.

·         It is not saved in Notify Messages page in the App.

·         How to redirect user to Notify Messages page after tapping on the
notification?


[image: index2.jpg]





Issue 2*: Error in cart*

This Error in cart after closing the app and reopen it again. And it still
exists even after upgrading to 1.6.6b



[image: index.jpg]



Thank you for your help and time.

Regards
